### PR TITLE
v-sokolov/shopXbootstrap

### PR DESCRIPTION
**Pull request by v-sokolov**
Implementing shopX-website  builded on Bootstrap 4

### CODE REVIEW

File: ShopeX/index.html [hunk 1]
```diff
@@ -0,0 +1,468 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>ShopeX v-sokolov </title>
+        <meta name="description">
```
Comment (volo-maslow) [inline ShopeX/index.html]: Please remove this meta tag.

File: ShopeX/index.html [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <!-- CSS Manual -->
+        <link rel="stylesheet" type="text/css" href="styles.css">
+        <link rel="stylesheet" type="text/css" href="./css/customStyles.css">
```
Comment (volo-maslow) [inline ShopeX/index.html]: Please, remove ./

File: ShopeX/index.html [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+        <!-- Fonts Google  Alex Brush/Geogrotesque/Roboto/Lato/Titilium Web -->
+        <link href="https://fonts.googleapis.com/css?family=Lato:300|Roboto:100,300,400,700|Titillium+Web:300,400,700&display=swap&subset=cyrillic"
+            rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Geo&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Alex+Brush&display=swap" rel="stylesheet">
+        <link href="http://allfont.ru/allfont.css?fonts=roboto-light" rel="stylesheet" type="text/css" />
+        <link href="http://allfont.ru/allfont.css?fonts=roboto-thin" rel="stylesheet" type="text/css" />
+            <!-- Specify in CSS:
```
Comment (volo-maslow) [inline ShopeX/index.html]: Please remove comments. You can put these comments to SASS.

File: ShopeX/index.html [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+            -->
+    </head>
+    <body>
+        <div class="header-background-line"></div>
+        <header class="container">
+            <div class="row logo-row">
+                <div class="header-search col-8 order-md-0 col-md-3">
+                    <form class="search-container">
+                        <button class="search-icon-button">
+                            <img src="./images/Search-Icon.png" class="search-icon">
```
Comment (volo-maslow) [inline ShopeX/index.html]: Please remove ./ everywhere.

File: ShopeX/index.html [hunk 5]
```diff
@@ -0,0 +1,468 @@
...
+                        </button>
+                        <input type="text" class="search" placeholder="search"/>
```
Comment (volo-maslow) [inline ShopeX/index.html:35]: Please add space before />

File: ShopeX/index.html [hunk 6]
```diff
@@ -0,0 +1,468 @@
...
+                    <a href="#"><img src="./images/dropdown-icon.png" class="dropdown"></a>
+                </div>
+                <div class="header-logo col-12 order-md-1 col-md-3">
+                        <img src="./images/logo.png" class="logo"/>
+                    </div>
+            </div>
+            <nav class="nav navbar navbar-expand-md">
+                <button class="navbar-toggler" type="button" data-toggle="collapse"
+                                                data-target="#navbarNav" aria-controls="navbarNav"
+                                                aria-expanded="false"
```
Comment (volo-maslow) [inline ShopeX/index.html]: Why do you need aria attributes here?

File: ShopeX/index.html [hunk 7]
```diff
@@ -0,0 +1,468 @@
...
+                                            <p class="name">Lava Vtase</p>
+                                            <p class="price">
+                                                <span>$65.00</span> $35.00
+                                            </p>
+                                        </div>
+                                        <div class="line-card">
+                                        </div>
+                                        <button class="button-card">
+                                            <img src="./images/basket-icon.png"/>
+                                            <a href="" >add to card</a>
```
Comment (volo-maslow) [inline ShopeX/index.html]: Remove useless space before >

File: ShopeX/css/main.scss [hunk 8]
```diff
@@ -0,0 +1,1037 @@
+@import "colors&fonts";
```
Comment (volo-maslow) [inline ShopeX/css/main.scss]: Using a & symbol could cause troubles with some systems, for example, Amazon Linux, so better to keep using only alphabet characters. I suggest removing this symbol to reject the change of troubles in the future.

File: ShopeX/css/main.scss [hunk 9]
```diff
@@ -0,0 +1,1037 @@
...
+.row {
+    width: 100%;
+}
+.header-background-line {
+    position: absolute;
+    top: 162px;
+    left: 0;
+    height: 1px;
+    width: 100%;
+    background-color: $headerBG;
```
Comment (volo-maslow) [inline ShopeX/css/main.scss]: Please, avoid shortcutting of variables, use headerBackground instead.

File: ShopeX/css/main.scss [hunk 10]
```diff
@@ -0,0 +1,1037 @@
...
+        }
+        .number a:hover {
+            color: $secondaryColor;
+        }
+        .white-pointer a{
+            color:#ffffff;
+            }
+        }
+}
+// SECTION 2 - CATALOG
```
Comment (volo-maslow) [inline ShopeX/css/main.scss]: You can split those sections into different files for better readability.

**Review by volo-maslow — CHANGES_REQUESTED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Pull request closed without merging**